modules/appnexusBidAdapter.js

```diff
@@ -265,12 +265,13 @@ export const spec = {
 
     if (bidderRequest && bidderRequest.refererInfo) {
       let refererinfo = {
-        rd_ref: encodeURIComponent(bidderRequest.refererInfo.referer),
+        // TODO: are these the correct referer values?
+        rd_ref: encodeURIComponent(bidderRequest.refererInfo.topmostLocation),
         rd_top: bidderRequest.refererInfo.reachedTop,
         rd_ifs: bidderRequest.refererInfo.numIframes,
         rd_stk: bidderRequest.refererInfo.stack.map((url) => encodeURIComponent(url)).join(',')
       }
-      let pubPageUrl = config.getConfig('pageUrl');
+      let pubPageUrl = bidderRequest.refererInfo.canonicalUrl;
```

Comment:
Our original use for `pageUrl` config option was for some custom AMP related scenarios where the normal refer page info was not available and when AMP caches it content; it would allow the publisher to ensure they were passing a value that made sense for the page.  I'd like to keep getConfig setup for now please.

Reply:
`canonicalUrl` picks `getConfig('pageUrl')`, then falls back to `<link rel="canonical">`. https://github.com/dgirardi/Prebid.js/blob/2ff990bbb19b78ee619468fe25c042e633fec34e/src/refererDetection.js#L84-L87

Reply:
Thanks for making the updates @dgirardi I think those would work out, so the changes here should be fine.